components/TruncatedAddress.tsx

```diff
@@ -10,13 +11,17 @@ const TruncatedAddress = ({
   sx = {},
   type,
   monoFont = true,
+  showDomain = false,
```

Comment:
Why set `showDomain` default `false`, domain will be displayed in most places because it's designed to replace an address in the presentation layer

Reply:
@Keith-CY Okay, 
I got it. I'll replace all the places if there exists a domain.